pkg/controllers/management/node/utils.go

```diff
@@ -130,6 +136,8 @@ func buildCommand(nodeDir string, node *v3.Node, cmdArgs []string) (*exec.Cmd, e
 		"PATH=/usr/bin:/var/lib/rancher/management-state/bin",
 	}
 	command.Env = jailer.WhitelistEnvvars(envvars)
+
+	logrus.Tracef("buildCommand args: %v", command.Args)
```

Comment:
Is just the args enough? Previous trace log was the whole thing

Reply:
its actually more, includes `rancher-machine` on the front. it's everything needed tho